ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -45,6 +45,16 @@ public protocol BadgeFieldDelegate: AnyObject {
     @objc optional func badgeFieldShouldReturn(_ badgeField: BadgeField) -> Bool
 }
 
+// MARK: - BadgeField Colors
+
+public extension Colors {
```

Comment:
Do these need to be public?

Reply:
it was public before so yes it has to be public for client apps to access these colors